.circleci/config.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - run:
           name: Run tests
           when: always
-          command: ./dev.py coverage
+          command: CI=True ./dev.py coverage 
```

Comment:
Shouldn't be necessary, I don't think

https://circleci.com/docs/variables/#built-in-environment-variables

Reply:
ah I see.